src/controller/tests/TestServerCommandDispatch.cpp

```diff
@@ -117,9 +120,8 @@ CHIP_ERROR TestClusterCommandHandler::EnumerateAcceptedCommands(const ConcreteCl
         return CHIP_NO_ERROR;
     }
 
-    // We just have one command id.
-    callback(Clusters::UnitTesting::Commands::TestSimpleArgumentRequest::Id, context);
-    return CHIP_NO_ERROR;
+    DataModel::AcceptedCommandEntry entries[] = { { TestSimpleArgumentRequest::Id, {} } };
```

Comment:
static constexpr? I am worried otherwise this may be a stack variable.

Reply:
I would even consider pulling this outside of the method. We really have to be careful to never return stack addresses.